test/evergreen.yml

```diff
@@ -2862,7 +2842,7 @@ buildvariants:
   display_name: "Large scale tests"
   batchtime: 480 # 3 times a day
   run_on:
-  - ubuntu1804-test
+  - ubuntu2004-test
```

Comment:
I thought there's a known issue of migrating large scale tests?

Reply:
Thanks for pointing out, I will change this.